src/pudl/metadata/fields.py

```diff
@@ -2456,6 +2509,14 @@
         ),
         "unit": "MW",
     },
+    "location": {
+        "type": "string",
+        "description": "Location of subsidiary company.",
+    },
```

Comment:
Do we have any analogous fields for this meaning already? Is this a freeform string? A place name? A fully specified address? Would it be clearer as `subsidary_location`? How does this field relate to / differ from `location_of_inc` below?

Reply:
I think `subsidiary_location` makes sense, and I've changed it. I think there's both state and country names in here, does that sound right @katie-lamb?

Reply:
Yeah, this is the location field that's pulled directly from the Ex. 21 so it's supposed to mean "location_of_incorporation" but it's not standardized whether that should be a state, city, country etc. So I think `subsidiary_location` is fine, or later we can update the Ex. 21 model to extract this column directly into `location_of_incorporation`.